src/QMCWaveFunctions/RotatedSPOs.cpp

```diff
@@ -511,8 +512,11 @@ void RotatedSPOs::exponentiate_antisym_matrix(ValueMatrix& mat)
   {
     for (int j = i; j < n; ++j)
     {
-      mat_h[i + n * j] = std::complex<RealType>(0, -1.0 * mat[j][i]);
-      mat_h[j + n * i] = std::complex<RealType>(0, 1.0 * mat[j][i]);
+      //This two liner does several things.  The first is it unpacks row-major mat into column major format.
+      //Second, it builds the hermitian matrix -i*mat.  Third, it relies on the hermiticity of -i*mat to
+      //fill the entire -i*mat matrix (in column major form) by iterating over the upper diagonal only.
+      mat_h[i + n * j] = std::complex<RealType>(std::imag(mat[i][j]), -1.0 * std::real(mat[i][j]));
+      mat_h[j + n * i] = std::complex<RealType>(-std::imag(mat[i][j]), 1.0 * std::real(mat[i][j]));
```

Comment:
I need to think through everything going on here. It looks like there is a relative transpose happening between what you're doing here vs. previously. Is this now consistent with what we've been working out in our notes?

Reply:
I assume by keeping it as mat[j][i] in the arguments, it produces the wrong matrix exponentials and logs in the tests?

Reply:
Actually, lets block this for now.  As it is written now, it passes the unit tests which were compared against scipy.linalg, whereas before it didn't (matrix was conjugated when it shouldn't have been).  However, lets make sure this doesn't break the real code finite difference tests of the derivative vs. parameter variations.

Reply:
Just checked it.  All good.